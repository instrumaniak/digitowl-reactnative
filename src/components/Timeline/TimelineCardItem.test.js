/**
 *   Tests for component: TimelineCardItem
 */

import React from 'react'
import TimelineCardItem from './TimelineCardItem'
import renderer from 'react-test-renderer'

describe('TimelineCardItem Component', () => {
  it('Renders correctly', () => {
    const tree = renderer.create(<TimelineCardItem />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
