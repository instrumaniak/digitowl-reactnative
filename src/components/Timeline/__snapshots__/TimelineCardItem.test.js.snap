// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimelineCardItem Component Renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#BBC8CA",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "width": 50,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 35,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingLeft": 15,
      }
    }
  >
    <View>
      <Text
        style={
          Object {
            "color": "#5B6267",
            "fontSize": 20,
          }
        }
      />
      <Text
        style={
          Object {
            "color": "#BBC8CA",
            "fontSize": 16,
          }
        }
      />
    </View>
    <View>
      <Text
        style={
          Object {
            "color": "#5B6267",
            "fontSize": 18,
            "fontWeight": "bold",
          }
        }
      />
    </View>
  </View>
</View>
`;
